themes/theme-b2b/src/components/Underlay.styles.ts

```diff
@@ -3,7 +3,7 @@ import { ThemeComponent, cva } from '@marigold/system';
 export const Underlay: ThemeComponent<'Underlay'> = cva('', {
   variants: {
     variant: {
-      modal: ' bg-bg-surface-underlay/40 backdrop-blur-sm',
+      modal: ' bg-bg-surface-sunken/40 backdrop-blur-sm',
```

Comment:
The underlay color in the story ("Overlay)" doesn't correspond to this value here.